### PR TITLE
First (simplest) attempt at parallax.

### PR DESCRIPTION
**Pull request by jobliz**
First attempt at solving #17. More images can be added / the image could be changed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,8 +61,9 @@
     "less": "^2.7.2",
     "lodash": "^4.17.4",
     "magnific-popup": "^1.1.0",
-    "npm-run-all": "^4.0.2",
     "nodemailer": "^4.0.1",
+    "npm-run-all": "^4.0.2",
+    "parallax-js": "git+https://github.com/wagerfield/parallax.git",
```
Comment (rivasvict) [inline package.json]: @jobliz Do we need this here?

**Review by rivasvict — CHANGES_REQUESTED** [review]
@jobliz please review the comment. Thanks.

**Comment by jobliz** [comment]
"parallax-js" was a leftover from tests that can be erased. I tried uninstalling npm-run-all, but then npm run serve-dev didn't run. I think the line was modified (extra space) when I added parallax-js and thus went into the changes. So, npm-run-all seems to be necessary.

File: src/client/app/landing.html [hunk 2]
```diff
@@ -90,6 +90,15 @@ <h2 class="section-heading">Nuestra historia</h2>
             </div>
         </div>
     </section>
+    
+    <div class="parallax-window"></div>
+    <script>
+      $(document).ready(function() {
```
Comment (rivasvict) [inline src/client/app/landing.html]: @jobliz Can you please move this logic inside a file propertly requested by webpack. Please make use of the fact that this page gets initialized here: https://github.com/vctr90/wedding-app/blob/development/src/client/app/mainAppLoader.js#L31 so you do not need the ready jquery function.

**Review by rivasvict — CHANGES_REQUESTED** [review]
@jobliz please take a look at the comments, Just those ones, other than that it looks good

**Review by rivasvict — APPROVED** [review]
@jobliz Thanks!! what a great Job *.* I loved it!!

**Pull request merged**